vision/src/autogluon/vision/predictor/predictor.py

```diff
@@ -541,6 +561,11 @@ def predict_proba(self, data, as_pandas=True):
         """
         if self._classifier is None:
             raise RuntimeError('Classifier is not initialized, try `fit` first.')
+        
+        if self._backend == 'automm':
+            prob = self._classifier.predict_proba(data=data, as_pandas=as_pandas)
```

Comment:
`predict_proba` is not implemented for OBJECT_DETECTION. What is expected to return for this function in detection task?

Reply:
We simply return a runtime error message saying that AutoMM doesn't support this functionality yet. Actually for image classification, we can't support every feature at this moment either.